src/Form/CitizenProjectMediaType.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AppBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class CitizenProjectMediaType extends AbstractType
```

Comment:
I think it should be abstract if the class is not meant to be used alone but only with class inheritance.

Reply:
the class may be used alone, why not